dali/pipeline/executor/executor.cc

```diff
@@ -467,8 +467,8 @@ void Executor::SetupDataForGraph(WorkspaceBlob *wsb) {
   }
 
   // Setup mixed op input and output buffers
-  for (int i = 0; i < graph_->NumMixedOp(); ++i) {
-    OpNode &node = graph_->mixed_node(i);
+  for (int i = 0; i < graph_->NumOp(OpType::MIXED); ++i) {
+    OpNode &node = graph_->Node(OpType::MIXED, i);
     MixedWorkspace &ws = wsb->mixed_op_data[i];
 
     // Mixed ops do not take argument tensors (at least for now)
```

Comment:
Maybe unrelated to this PR but inportant for new nvJPEG fused crop operators:
Can we make mixed operators accept argument tensors as well?